pimd/pimd.h

```diff
@@ -79,6 +79,8 @@
 #define MCAST_ALL_PIM_ROUTERS  "224.0.0.13"
 #define MCAST_ALL_IGMP_ROUTERS "224.0.0.22"
 
+#define MCAST_ALLRTRS_GROUP 0xe0000002U /* 224.0.0.2 */
```

Comment:
sharpd@eva ~/frr7 (tests_dict_values) [1]> git grep "224.0.0.2"
ldpd/ldp.h:#define AllRouters_v4                "224.0.0.2"
pimd/pimd.h:#define MCAST_ALL_ROUTERS      "224.0.0.2"
pimd/pimd.h:#define MCAST_ALL_IGMP_ROUTERS "224.0.0.22"
zebra/irdp.h:#define INADDR_ALLRTRS_GROUP  0xe0000002U    /* 224.0.0.2 */
sharpd@eva ~/frr7 (tests_dict_values)> 

let's get this consistent across everything instead of adding another one off define